sql/updates/world/6.x/2016_05_13_00_world.sql

```diff
@@ -0,0 +1,742 @@
+(43679, 16, 0, 'The Alliance has lost control of Warden''s Vigil!', 41, 0, 100, 0, 0, 0, 45470, 2, ''),
+(43679, 17, 0, 'The Horde has taken Warden''s Vigil!', 41, 0, 100, 0, 0, 0, 45472, 2, ''),
+(43679, 18, 0, 'The Horde has lost Warden''s Vigil!', 41, 0, 100, 0, 0, 0, 45473, 2, ''),
+(43679, 19, 0, 'The Alliance has successfully defended Tol Barad!', 41, 0, 100, 0, 0, 8455, 45174, 2, ''),
+(43679, 20, 0, 'The Horde has successfully defended Tol Barad!', 41, 0, 100, 0, 0, 8454, 45188, 2, ''),
+(43679, 21, 0, 'The Alliance has successfully taken Tol Barad!', 41, 0, 100, 0, 0, 8455, 45189, 2, ''),
+(43679, 22, 0, 'The Horde has successfully taken Tol Barad!', 41, 0, 100, 0, 0, 8454, 45190, 2, ''),
+(43679, 23, 0, 'Warning: This vehicle is not permitted to operate outside of Tol Barad. Turn back now.', 41, 0, 100, 0, 0, 0, 45526, 0, ''),
+(43679, 24, 0, 'Preparations for Tol Barad will begin in 5 minutes!', 41, 0, 100, 0, 0, 0, 50887, 2, ''),
+(43679, 25, 0, 'Preparations for Tol Barad will begin in 2 minutes!', 41, 0, 100, 0, 0, 0, 50888, 2, ''),
+(43679, 26, 0, 'Preparations for Tol Barad will begin in 1 minute!', 41, 0, 100, 0, 0, 0, 50889, 2, ''),
+(47324, 1, 0, 'Victory in Tol Barad is ours! While the Horde offensive is crippled, we need your immediate efforts to gather resources and secure the island!', 41, 0, 100, 0, 0, 8455, 50894, 0, ''),
+(47335, 1, 0, 'Victory in Tol Barad is ours! While the Alliance offensive is crippled, we need your immediate efforts to gather resources and secure the island!', 41, 0, 100, 0, 0, 8455, 50895, 0, '');
+
+-- GRAVEYARDS -- 
+-- Spirit Guides (45066 - 45079)
+UPDATE `creature_template` SET `gossip_menu_id` = 45068, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45068;
+UPDATE `creature_template` SET `gossip_menu_id` = 45069, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45069;
+UPDATE `creature_template` SET `gossip_menu_id` = 45070, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45070;
+UPDATE `creature_template` SET `gossip_menu_id` = 45071, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45071;
+UPDATE `creature_template` SET `gossip_menu_id` = 45072, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45072;
+UPDATE `creature_template` SET `gossip_menu_id` = 45073, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45073;
+UPDATE `creature_template` SET `gossip_menu_id` = 45068, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45074;
+UPDATE `creature_template` SET `gossip_menu_id` = 45069, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45075;
+UPDATE `creature_template` SET `gossip_menu_id` = 45073, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45076;
+UPDATE `creature_template` SET `gossip_menu_id` = 45072, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45077;
+UPDATE `creature_template` SET `gossip_menu_id` = 45070, `ScriptName` = 'npc_tb_spirit_guide' WHERE `entry` = 45078;
```

Comment:
This list of updates can be made 50% shorter in terms of line numbers, like this:
```sql
-- GRAVEYARDS -- 
-- Spirit Guides (45066 - 45079)
UPDATE `creature_template` SET `gossip_menu_id`= 45068, `ScriptName`= 'npc_tb_spirit_guide' WHERE `entry` IN (45068,45074);
UPDATE `creature_template` SET `gossip_menu_id`= 45069, `ScriptName`= 'npc_tb_spirit_guide' WHERE `entry` IN (45069,45075);
UPDATE `creature_template` SET `gossip_menu_id`= 45070, `ScriptName`= 'npc_tb_spirit_guide' WHERE `entry` IN (45070,45078);
UPDATE `creature_template` SET `gossip_menu_id`= 45071, `ScriptName`= 'npc_tb_spirit_guide' WHERE `entry` IN (45071,45079);
UPDATE `creature_template` SET `gossip_menu_id`= 45072, `ScriptName`= 'npc_tb_spirit_guide' WHERE `entry` IN (45072,45077);
UPDATE `creature_template` SET `gossip_menu_id`= 45073, `ScriptName`= 'npc_tb_spirit_guide' WHERE `entry` IN (45073,45076);
```
(I haven't had time to check the rest of your SQL for redundancy yet)